db/schema.sql

```diff
@@ -0,0 +1,13 @@
+DROP TABLE  IF EXISTS users;
+
+    CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    age TEXT,
+    name TEXT,
+    company TEXT NOT NULL UNIQUE,
+    country TEXT,
```

Comment:
Why did you decide to make `company` unique? This constraint means that we cannot have any users that work for the same company and I think that would cause a problem in real life. I'm also not sure about the unique constraint for photo. If you have millions of uses, some might want to use the same photo url and that should be okay.